src/App.tsx

```diff
@@ -1,8 +1,90 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+import { TodoWithUser, loadTodos, loadUsers } from './utils';
+import { TodoList } from './components/TodoList';
+
+const App: FC<{}> = () => {
```

Comment:
в функциях пустые пропсы не указываются
```suggestion
const App: FC = () => {
```